public/app/features/explore/TimeSyncButton.tsx

```diff
@@ -18,15 +17,12 @@ export function TimeSyncButton(props: TimeSyncButtonProps) {
 
   return (
     <Tooltip content={syncTimesTooltip} placement="bottom">
-      <button
-        className={classNames('btn navbar-button navbar-button--attached', {
-          [`explore-active-button`]: isSynced,
-        })}
+      <ToolbarButton
+        icon="link"
+        active={isSynced}
         aria-label={isSynced ? 'Synced times' : 'Unsynced times'}
         onClick={() => onClick()}
```

Comment:
```suggestion
onClick={onClick}
```
No need to create a new function on every render here.